geode-core/src/main/java/org/apache/geode/internal/cache/DiskStoreImpl.java

```diff
@@ -4564,4 +4569,13 @@ public StatisticsFactory getStatisticsFactory() {
     return this.cache.getDistributedSystem();
   }
 
+  public int getDiskUsagePercentage() {
+    long totalDiskSpace = 0;
+    for (DirectoryHolder dr : this.directories) {
+      totalDiskSpace += dr.getDiskDirectoryStats().getDiskSpace();
+    }
```

Comment:
The same calculation to get the used disk space is done in `DiskStoreMBeanBridge.getTotalBytesOnDisk()`. I recommend either moving this method into `DiskStoreMBeanBridge` or moving `getTotalBytesOnDisk()` into `DiskStoreImpl` so that the code can be reused.

Reply:
Right, its better to reuse code. I think it fits better in DiskStoreImpl, so the bridge just call the implementation of its diskStore object, as it does with other methods.